api/grist.yml

```
@@ -443,6 +443,18 @@ paths:
             text/json:
               schema:
                 $ref: '#/components/schemas/TableSchemaResult'
+  /docs/{docId}/force-reload:
+    post:
+      operationId: forceReload
+      tags:
+        - docs
+      summary: "Reload a document"
+      description: "Close and reopen the document, forcing the python engine to retart."
```

May be conjugate the description at the third person as a *it*
```suggestion
      description: "Closes and reopens the document, forcing the python engine to restart."
```